### ISSUE TITLE
Support for multi-line text

### ISSUE THREAD

**Issue by nvlashi** [justkawal/animated_text#1]
First off, great work!  This is a really cool library. 

I was trying to use this with multiline text and hoping to use it for animations for doing blocks of text. When I set the word list array to an array of strings which contain newline (\n), it is not breaking on the new line character. I was looking at the source to see if I could figure out where the issue was but could not identify. it.

Any chance to add this support?

**Comment by justkawal** [comment]
Yes I had thought to add this functionality in the coming version, but as for now I am busy with excel password protection functionality, in [excel](https://github.com/justkawal/excel)

So I thinks that it might take a while.
Sorry for the inconvenience occurred to u.

I hope u understand!

**Comment by nvlashi** [comment]
> Yes I had thought to add this functionality in the coming version, but as for now I am busy with excel password protection functionality, in [excel](https://github.com/justkawal/excel)
> 
> So I thinks that it might take a while.
> Sorry for the inconvenience occurred to u.
> 
> I hope u understand!

Of course.  I look forward to your update.
